dw-action-toolbar.js

```diff
@@ -233,80 +233,84 @@ export class DwActionToolbar extends LitElement {
 
   set primaryActions(val) {
     let oldValue = this.__primaryActions;
-    if(isEqual(oldValue, val)) {
+    if (isEqual(oldValue, val)) {
       return;
     }
 
     this.__primaryActions = val;
-    this.requestUpdate('primaryActions', oldValue);
+    this.requestUpdate("primaryActions", oldValue);
     this._computeItems();
   }
 
   constructor() {
     super();
     this.singleSelect = true;
-    this.triggerIcon = 'more_vert';
+    this.triggerIcon = "more_vert";
     this.primaryActions = [];
     this._primaryActions = [];
     this._secondaryActions = [];
     this.primaryActionButtonSize = 48;
     this.primaryActionIconSize = 24;
     this.listItemIconSize = 24;
-    this.closeIcon = 'close';
-    this.closeIconPosition = 'right';
-    this.dialogHAlign = 'left';
-    this.dialogVAlign = 'bottom';
+    this.closeIcon = "close";
+    this.closeIconPosition = "right";
+    this.dialogHAlign = "left";
+    this.dialogVAlign = "bottom";
   }
 
   render() {
     return html`
-      ${this._primaryActions && this._primaryActions.length ? html`
-          ${repeat(this._primaryActions, (action) => action.name, (action, index) => html`
-          <dw-icon-button
-            class="primary-action-btn"
-            style="${styleMap(this._setPrimaryActionIconColor(action))}"
-            .iconSize="${this.primaryActionIconSize}"
-            .buttonSize="${this.primaryActionButtonSize}"
-            .title="${action.tooltip ? action.tooltip : ''}"
-            name="${action.name}"
-            icon="${action.icon}"
-            @click=${this._onPrimaryActionClick}>
-          </dw-icon-button>
-          `)}
-        ` : ''}
-        ${this._secondaryActions && this._secondaryActions.length ? html`
-          <dw-select
-            ?custom-trigger="${this.customTrigger}"
-            .opened="${this.opened}"
-            .singleSelect="${true}"
-            .triggerIcon="${this.triggerIcon}"
-            .triggerButtonSize="${this.primaryActionButtonSize}"
-            .triggerIconSize="${this.primaryActionIconSize}"
-            .backIcon="${this.closeIcon}"
-            .backIconPosition="${this.closeIconPosition}"
-            .noCloseIcon="${this.noCloseIcon}"
-            .items="${this._secondaryActions}"
-            .disabledItems="${this.disabledActions}"
-            .value="${this._value}"
-            .listItemIconSize=${this.listItemIconSize}
-            .itemLabel="${"label"}"
-            .itemValue="${"name"}"
-            .mobileMode="${this.mobileMode}"
-            .noHeader="${this.noHeader}"
-            .dialogTitle="${this.dialogTitle}"
-            .vAlign="${this.dialogVAlign}"
-            .hAlign="${this.dialogHAlign}"
-            .vOffset="${this.dialogVOffset}"
-            .hOffset="${this.dialogHOffset}"
-            .alwaysFullScreenInMobile="${this.alwaysFullScreenInMobile}"
-            @value-changed="${this._triggerActionEvent}"
-            @opened-changed="${this._onSelectOpenedChanged}"
```

Comment:
`_onSelectOpenedChanged` handler isn't removed from the code; though it's not used anymore.